Sources/SATSCore/Components/DatePickerView/DatePickerView-Shared.swift

```diff
@@ -0,0 +1,77 @@
+import SwiftUI
+
+extension DatePickerView {
```

Comment:
if you want to test this stuff, I guess you would need a value where you can inject `Calendar.current` and call methods on it.

From the DatePickerView perspective, it can just create a value like that can the methods will call the object's methods.

That's only for testability